File: backend/app/Models/SalesReturnItem.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class SalesReturnItem extends Model
{
    use HasFactory;

    protected $fillable = [
        'sales_return_id',
        'batch_id',
        'quantity',
        'selling_price',
        'total',
    ];

    public function salesReturn()
    {
        return $this->belongsTo(SalesReturn::class);
    }

    public function batch()
    {
        return $this->belongsTo(Batch::class);
    }
}

File: desktop/lib/features/sales/domain/entities/sales_return_item.dart
class SalesReturnItem {
  final String id;
  final String salesReturnId;
  final String batchId;
  final String? batchNumber;
  final String? medicineName;
  final int quantity;
  final double sellingPrice;
  final double total;

  const SalesReturnItem({
    required this.id,
    required this.salesReturnId,
    required this.batchId,
    this.batchNumber,
    this.medicineName,
    required this.quantity,
    required this.sellingPrice,
    required this.total,
  });
}
